index.css

```diff
@@ -152,6 +163,193 @@ li {
   align-items: center;
 }
 
+.textside p {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills li {
+  background-color: #ebebff;
+  padding: 6px 5px;
+  border-radius: 8px;
+  font-size: 15px;
+  color: #396df2;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.about-skills li {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  padding: 20px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  width: 140px;
+}
+
+.card-ul li {
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #6b778c;
+}
+
+.card-ul li:nth-child(1) {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills {
+  text-align: center;
+  display: flex;
+}
+
+.see-project {
+  background-color: #396df2;
+  color: #fff;
+  border: none;
+  border-radius: 9px;
+  padding: 5px 8px;
+  align-self: flex-start;
+  letter-spacing: 0.03em;
+  width: 140px;
+  height: 50px;
+  font-size: 15px;
+  font-weight: 700;
+}
```

Comment:
- [ ] Wonderful job so far. However, there's little thing you need to work on here. The background color for the enable state of your  button is different from what we have on the Figma design. Kindly take a closer look to both designs and work your design alongside the Figma design. 

### Your Design
![your-button-bg-color](https://user-images.githubusercontent.com/26597432/220644310-96f5d6ca-46a1-477e-84d2-e6499d5b41d0.png)

### Figma Design
![temp-button-bg-color](https://user-images.githubusercontent.com/26597432/220644363-cb7ca88e-780b-43e7-a29c-1cc71b68221a.png)


![buttons](https://user-images.githubusercontent.com/26597432/220644399-75d10596-59db-4979-81e0-6fa69f4c4e83.png)

Reply:
Fixed